repos.csv

```diff
@@ -1,9868 +1,9892 @@
-jenkinsci/environment-variable-page-decoration-plugin,master,8,,gradlew
-jenkinsci/escaped-markup-plugin,master,8,,maven
-jenkinsci/evergreen-plugin,master,8,,maven
-jenkinsci/exam-plugin,develop,8,,maven
-jenkinsci/exclusion-plugin,master,8,,maven
-jenkinsci/exclusive-execution-plugin,master,8,,maven
-jenkinsci/extended-choice-parameter-plugin,master,8,,maven
-jenkinsci/extended-read-permission-plugin,master,8,,maven
-jenkinsci/extended-security-settings-plugin,master,8,,maven
-jenkinsci/extensible-choice-parameter-plugin,master,8,,maven
-jenkinsci/extension-filter-plugin,master,8,,maven
-jenkinsci/extensivetesting-plugin,master,8,,maven
-jenkinsci/external-logging-api-plugin,master,8,,maven
-jenkinsci/external-logging-elasticsearch-plugin,master,8,,maven
-jenkinsci/external-monitor-job-plugin,master,8,,maven
-jenkinsci/external-resource-dispatcher-plugin,master,8,,maven
-jenkinsci/external-scheduler-plugin,master,8,,maven
-jenkinsci/external-workspace-manager-plugin,master,8,,maven
-jenkinsci/extras-client-demo,master,8,,maven
-jenkinsci/extras-ec2-launcher,master,8,,maven
-jenkinsci/extras-executable-war,master,8,,maven
-jenkinsci/extras-file-change-monitor,master,8,,maven
-jenkinsci/extras-jenkins-wiki-plugin,master,8,,maven
-jenkinsci/extras-jinterop-wmi,master,8,,maven
-jenkinsci/extras-maven2.1-interceptor,master,8,,maven
-jenkinsci/extras-memory-monitor,master,8,,maven
-jenkinsci/extras-nofinal,master,8,,maven
-jenkinsci/extras-nofinal-test,master,8,,maven
-jenkinsci/extras-tester,master,8,,maven
-jenkinsci/fabric-beta-publisher-plugin,master,8,,maven3.3.9
-jenkinsci/failure-visualizer-plugin,master,8,,maven
-jenkinsci/fast-track-plugin,master,8,,gradlew
-jenkinsci/favorite-plugin,master,8,,maven
-jenkinsci/favorite-view-plugin,master,8,,maven
-jenkinsci/fedora-module-build-system-plugin,master,8,,maven
-jenkinsci/figlet-buildstep-plugin,master,8,,maven
-jenkinsci/file-leak-detector-plugin,master,8,,maven
-jenkinsci/file-operations-plugin,master,8,,maven
-jenkinsci/file-parameters-plugin,master,8,,maven
-jenkinsci/files-found-trigger-plugin,master,8,,maven
-jenkinsci/filesystem-list-parameter-plugin,master,8,,maven
-jenkinsci/filesystem_scm-plugin,master,8,,maven
-jenkinsci/firefox-extension-buildmonitor,master,8,,maven
-jenkinsci/fit-plugin,master,8,,maven
-jenkinsci/fitnesse-plugin,master,8,,maven
-jenkinsci/flaky-test-handler-plugin,master,8,,maven
-jenkinsci/flexteam-plugin,master,8,,maven
-jenkinsci/flyway-runner-plugin,master,8,,maven3.3.9
-jenkinsci/flyweight-build-step-plugin,master,8,,maven
-jenkinsci/fogbugz-plugin,master,8,,maven
-jenkinsci/folder-auth-plugin,master,8,,maven
-jenkinsci/font-awesome-api-plugin,master,11,,maven
-jenkinsci/font-awesome-icons-plugin,master,11,,maven
-jenkinsci/forensics-api-plugin,master,11,,maven
-jenkinsci/form-element-path-plugin,master,8,,maven
-jenkinsci/forticwp-cicd-plugin,main,8,,maven
-jenkinsci/fortify-cloudscan-plugin,master,8,,maven
-jenkinsci/fortify-on-demand-uploader-plugin,m
```

Comment:
The projects past 9869 may already exist and/or are in the wrong locations with the wrong format.